.github/workflows/build-and-test.yml

```diff
@@ -83,8 +88,36 @@ jobs:
                   python -m pip install -r requirements.txt -r requirements-dev.txt
                   python setup.py install
 
-            - name: Run model tests
-              run: make test
+            - name: Set up MacOS python
+              if: matrix.os != 'windows-latest'
+              uses: goanpeca/setup-miniconda@v1.1.2
+              with:
+                  activate-environment: env
+                  auto-update-conda: true 
+                  python-version: ${{ matrix.python-version }}
+                  channels: conda-forge, anaconda
+            
+            - name: Install dependencies MacOS
+              if: matrix.os != 'windows-latest'
+              shell: bash -l {0}
+              run: |
+                  conda upgrade -y pip setuptools
+                  conda install Cython requests numpy
+                  python ./scripts/convert-requirements-to-conda-yml.py requirements.txt requirements-dev.txt requirements-gui.txt > requirements-all.yml
+                  conda env update --file requirements-all.yml
+                  python setup.py install
```

Comment:
Here, `python` is the conda environment python. But see below, that's not true in the next section.